src/core/Akka.Remote/EndpointManager.cs

```diff
@@ -340,44 +373,62 @@ protected override SupervisorStrategy SupervisorStrategy()
                 var directive = Directive.Stop;
 
                 ex.Match()
-                    .With<InvalidAddressAssociationException>(ia =>
```

Comment:
Are there any cases where the exceptions might be part of a remoting message?
e.g. an exception on a remote node where the exception might be propagated over in a supervision message to another node for example?
Just thinking about serializability and visibility of the exception type.. might be a non issue